packages/server/lib/controllers/v1/connection/get.ts

```diff
@@ -71,7 +71,7 @@ export const getConnection = asyncWrapper<GetConnection>(async (req, res) => {
 
             res.status(400).send({
                 errorLog,
-                provider: null,
+                provider: null, // TODO: fix this
```

Comment:
Just making sure you're aware of this `TODO`

Reply:
Yep, it's a technical problem deeper than this but I found out during this PR https://linear.app/nango/issue/NAN-1683/when-a-credentials-refresh-fails-getconnection-is-failing-without-a